### DISCUSSION TITLE
Update documentation with context receivers section?

### DISCUSSION THREAD

**Discussion by qwexter** [arrow-kt/arrow#3479] [General]
Hi! I've just learned that the context receivers feature will be [phased out soon](https://kotlinlang.org/docs/whatsnew-eap.html#phased-removal-of-context-receivers-feature). Additionally, I found a [new proposal](https://github.com/Kotlin/KEEP/blob/context-parameters/proposals/context-parameters.md) by @serras.

What do you think about removing all mentions of context receivers from the Arrow documentation? It might be misleading to suggest using this feature now, I suppose.

@serras @nomisRev

**Comment by serras** [comment]
Such a migration is scheduled, and some of the docs have been updated. However, we're waiting for context parameters to appear in order to have examples using that feature.
